src/sections/upload-dataset-files/FileUploader.tsx

```diff
@@ -0,0 +1,187 @@
+import { ChangeEventHandler, DragEventHandler, useEffect, useState } from 'react'
+import { ProgressBar, Card } from 'react-bootstrap'
```

Comment:
We try not to use react-bootstrap components directly from SPA, we always try to import them from our design system.
So if you need a react-bootstrap component that is not yet in the design system, you should re-export it from the design system, apply the props interface you see necessary so we don't depend on the library interface itself, add a story to storybook, a basic unit test and document the new addition in the CHANGELOG.md under # Non Published Changes.
This way if in the future we change the component library used in the design system we are not so coupled to it from the SPA.
If you need further explanation on this, please let me know or we can have a quick pair progamming.